### PR TITLE
Social - make chirp like/dislike exclusive

### PR DESCRIPTION
**Pull request by dayan9265**
These changes makes that if a user likes a chirp that he previously had disliked, it will remove the dislike reaction (same thing for a liked chirp). This small change avoids the situation of a chirp being liked and disliked which in theory doesn't make a lot of sense.

### CODE REVIEW

**Review by MeKHell — CHANGES_REQUESTED** [review]

**Review by MeKHell — CHANGES_REQUESTED** [review]
Simple refactoring needed.

File: fe1-web/src/features/social/components/ChirpCard.tsx [hunk 1]
```diff
@@ -155,6 +155,26 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
 
   const [showDeleteConfirmation, setShowDeleteConfirmation] = useState(false);
 
+  const thumbsPressed = (isThumbsUp: boolean) => {
+    const reactionToAdd = isThumbsUp ? '👍' : '👎';
+    const reactionToRemove = isThumbsUp ? '👎' : '👍';
```
Comment (Tyratox) [inline fe1-web/src/features/social/components/ChirpCard.tsx:160]: What about

```suggestion
  const thumbsPressed = (reactionToAdd: '👍' | '👎') => {
    const reactionToRemove = isThumbsUp ? '👎' : '👍';
```

File: fe1-web/src/features/social/components/ChirpCard.tsx [hunk 2]
```diff
@@ -155,6 +155,26 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
...
+    // If the user has already reacted with the other reaction, remove it
+    if (reacted[reactionToRemove]) {
+      deleteReaction(reacted[reactionToRemove].id);
+    }
+
+    addReaction(reactionToAdd);
+  };
+
+  const thumbsUpPressed = () => thumbsPressed(true);
+  const thumbsDownPressed = () => thumbsPressed(false);
```
Comment (Tyratox) [inline fe1-web/src/features/social/components/ChirpCard.tsx:176]: If you need to pass an argument to a function feel free to write

`onPress={() => thumbsPressed("👍")}`

in the component directly. If there are no arguments, `onPress={someFunction}` is usually preferred.

**Review by MeKHell — APPROVED** [review]
This looks good to me

**Review by Tyratox — APPROVED** [review]
Looks good to me as well 😊

**Pull request merged**